javascript/browser-apis/toolbox-i/battery-api.md

```diff
@@ -70,7 +70,7 @@ navigator.???()
 - getBattery
 - battery
 - addEventListener
-- 'levelChange'
+- 'levelchange'
```

Comment:
should we add one more event name that's wrong?

Reply:
Oh Did not see this. Sure